crates/e2e/tests/e2e/protocol_fee.rs

```diff
@@ -68,166 +38,7 @@ async fn local_node_price_improvement_fee_sell_order() {
-    )
-    .await;
-}
-
-async fn surplus_fee_sell_order_capped_test(web3: Web3) {
-    let fee_policy = FeePolicyKind::Surplus {
-        factor: 0.9,
-        max_volume_factor: 0.1,
-    };
-    let protocol_fees = vec![
-        ProtocolFee {
-            policy: fee_policy.clone(),
-            policy_order_class: FeePolicyOrderClass::Market,
-        },
-        // Tests ability of providing multiple fees in the config. In fact, the Market one is used
-        // only since the order is in-market.
-        ProtocolFee {
-            policy: fee_policy,
-            policy_order_class: FeePolicyOrderClass::Limit,
-        },
-    ];
-    // Without protocol fee:
-    // Expected execution is 10000000000000000000 GNO for
-    // 9871415430342266811 DAI, with executed_surplus_fee = 167058994203399 GNO
-    //
-    // With protocol fee:
-    // Expected executed_surplus_fee is 167058994203399 +
-    // 0.1*(10000000000000000000 - 167058994203399) = 1000150353094783059
-    //
-    // Final execution is 10000000000000000000 GNO for 8884273887308040129 DAI, with
-    // executed_surplus_fee = 1000150353094783059 GNO
-    //
-    // Settlement contract balance after execution = 1000150353094783059 GNO =
-    // 1000150353094783059 GNO * 8884273887308040129 / (10000000000000000000 -
-    // 1000150353094783059) = 987306456662572858 DAI
-    execute_test(
-        web3.clone(),
-        vec![ProtocolFeesConfig(protocol_fees)],
-        OrderKind::Sell,
-        None,
-        1000150353094783059u128.into(),
-        987306456662572858u128.into(),
-    )
-    .await;
-}
-
-async fn volume_fee_sell_order_test(web3: Web3) {
-    let fee_policy = FeePolicyKind::Volume { factor: 0.1 };
-    // Tests ability of providing `any` fee policy order class config. In fact, the
-    // Market config could be used since the order is in-market.
-    let protocol_fee = ProtocolFee {
-        policy: fee_policy,
-        policy_order_class: FeePolicyOrderClass::Any,
-    };
-    // Without protocol fee:
-    // Expected execution is 10000000000000000000 GNO for
-    // 9871415430342266811 DAI, with executed_surplus_fee = 167058994203399 GNO
-    //
-    // With protocol fee:
-    // Expected executed_surplus_fee is 167058994203399 +
-    // 0.1*(10000000000000000000 - 167058994203399) = 1000150353094783059
-    //
-    // Final execution is 10000000000000000000 GNO for 8884273887308040129 DAI, with
-    // executed_surplus_fee = 1000150353094783059 GNO
-    //
-    // Settlement contract balance after execution = 1000150353094783059 GNO =
-    // 1000150353094783059 GNO * 8884273887308040129 / (10000000000000000000 -
-    // 1000150353094783059) = 987306456662572858 DAI
-    execute_test(
-        web3.clone(),
-        vec![ProtocolFeesConfig(vec![protocol_fee])],
-        OrderKind::Sell,
-        None,
-        1000150353094783059u128.into(),
-        987306456662572858u128.into(),
-    )
-    .await;
-}
-
 async fn partner_fee_sell_order_test(web3: Web3) {
```

Comment:
We do not test the partner fees in any other place. Please, do not delete this test (neither the one below). There are the only tests using the `appData` in order to test the partner fee.